lib/usecases/getSecureUploadUrl.js

```diff
@@ -0,0 +1,15 @@
+import { nanoid } from 'nanoid';
+import createGateway from '../gateways/s3';
+import s3config from '../s3config';
+
+const documents = createGateway({
+  ...s3config,
+  configuration: { urlPrefix: process.env.URL_PREFIX, maxUploadBytes: 20971520 }
+});
+
+export default async dropboxId => {
```

Comment:
As per @bjpirt 's comments. The dropbox code can be deleted.

Reply:
The dropboxId in this context is just the name of the folder in s3. There is still ongoing discussion on how to use folders.

Reply:
This is the code which generates the secureurl